Index: src/main/resources/templates/TokenMgrError.template
===================================================================
--- src/main/resources/templates/TokenMgrError.template	(revision 765)
+++ src/main/resources/templates/TokenMgrError.template	(working copy)
@@ -103,7 +103,7 @@
     return("Lexical error at line " +
           errorLine + ", column " +
           errorColumn + ".  Encountered: " +
-          (EOFSeen ? "<EOF> " : ("\"" + addEscapes(String.valueOf(curChar1)) + "\"") + " (" + (int)curChar + "), ") +
+          (EOFSeen ? "<EOF> " : ("\"" + addEscapes(String.valueOf(curChar1)) + "\"") + " (" + curChar + "), ") +
           "after : \"" + addEscapes(errorAfter) + "\"");
   }
 
